### PR TITLE
New article for GitBucket 4.8 release

### PR DESCRIPTION
**Pull request by takezoe**

### CODE REVIEW

File: _posts/2016-12-23-gitbucket-4.8.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Today, we are pleased to announce the availability of new version of Scala based Git platform [GitBucket 4.8](https://github.com/gitbucket/gitbucket/releases/tag/4.8)!
+
+## Search for repository names
+
+The search box on the global header now works for search repository:
+
+![Repository search box on the header]({{site.baseurl}}/images/gitbucket-4.8/repository_search_box.png)
+
+Also the filter box on the dashboard sidebar to filter repositories has been available:
```
Comment (McFoggy) [inline _posts/2016-12-23-gitbucket-4.8.markdown]: would have formulated `Also a filter box on the dashboard sidebar to filter repositories have been introduced:`

File: _posts/2016-12-23-gitbucket-4.8.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Search issues and wiki
+
+The search box has been available for issues and wiki:
+
+![Search box for issues]({{site.baseurl}}/images/gitbucket-4.8/search_box_issues.png)
+
+![Search box for wiki]({{site.baseurl}}/images/gitbucket-4.7/search_box_wiki_box.png)
+
+In addition, this release contains new WebAPI to get a single issue, performance improvenemt for the repository viewer. It also contains some improvements and bug fix. See [all closed issues in this release](https://github.com/gitbucket/gitbucket/issues?q=is%3Aclosed+milestone%3A4.8) to know details.
```
Comment (McFoggy) [inline _posts/2016-12-23-gitbucket-4.8.markdown]: split items in list ?

------- 

In addition, this release contains:

- new REST API endpoint to get a single issue
- performance improvenemt for the repository viewer
- improvements and bug fix, see [all closed issues in this release](https://github.com/gitbucket/gitbucket/issues?q=is%3Aclosed+milestone%3A4.8)

**Comment by takezoe** [comment]
@McFoggy Thanks :-) I will add some more topics and screen shot before publishing, so I hope you review it again.

File: _posts/2016-12-23-gitbucket-4.8.markdown [hunk 3]
```diff
@@ -24,8 +24,20 @@ The search box has been available for issues and wiki:
 
 ![Search box for issues]({{site.baseurl}}/images/gitbucket-4.8/search_box_issues.png)
 
-![Search box for wiki]({{site.baseurl}}/images/gitbucket-4.7/search_box_wiki_box.png)
+![Search box for wiki]({{site.baseurl}}/images/gitbucket-4.8/search_box_wiki.png)
 
-In addition, this release contains new WebAPI to get a single issue, performance improvenemt for the repository viewer. It also contains some improvements and bug fix. See [all closed issues in this release](https://github.com/gitbucket/gitbucket/issues?q=is%3Aclosed+milestone%3A4.8) to know details.
+## Keep pull request comments
+
+Before this release, comments on source code of the pull request had became invisible if new commits are pushed. Since this release, if new pushed commits don't update commented line, these comments are keeped on the screen.
```
Comment (McFoggy) [inline _posts/2016-12-23-gitbucket-4.8.markdown]: are ~keeped~ kept

File: _posts/2016-12-23-gitbucket-4.8.markdown [hunk 4]
```diff
@@ -24,8 +24,20 @@ The search box has been available for issues and wiki:
...
+
+![Comment for the pull request]({{site.baseurl}}/images/gitbucket-4.8/pull_request_comment.png)
+
+In addition, this release contains:
+
+- New WebAPI to get a single issue
```
Comment (McFoggy) [inline _posts/2016-12-23-gitbucket-4.8.markdown]: WebAPI = >`[WebAPI](https://github.com/gitbucket/gitbucket/wiki/API-WebHook#api)`

File: _posts/2016-12-23-gitbucket-4.8.markdown [hunk 5]
```diff
@@ -24,8 +24,20 @@ The search box has been available for issues and wiki:
...
+- Performance improvement for the repository viewer
+- Some other improvements and bug fix
```
Comment (McFoggy) [inline _posts/2016-12-23-gitbucket-4.8.markdown]: ~bug fix~ => bugfixes

wasn't sure of this, taken from [wiktionary](https://fr.wiktionary.org/wiki/bugfix)

**Review by McFoggy — APPROVED** [review]
looks good

**Comment by McFoggy** [comment]
http://gitbucket-community.herokuapp.com/ is now up to date with 4.8 deployed

**Comment by takezoe** [comment]
@McFoggy Great. Thanks!

**Pull request merged**